styles.css

```diff
@@ -489,6 +489,104 @@ label {
   display: none;
 }
 
+.popup {
+  position: fixed;
+  top: 0;
+  left: 0;
+  min-width: 100%;
+  min-height: 100%;
+  z-index: 999;
+  padding-top: 1%;
+  background-color: rgba(193, 199, 208, 0.8);
+  backdrop-filter: blur(7px) brightness(1);
+}
+
+.popup_content {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 75%;
+  height: 90%;
+  background-color: white;
+  padding: 20px;
+  border-radius: 10px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.5);
+}
```

Comment:
- [ ] You have done a great job. Allow me to say congratulations 🎆 👏 . The popup window is working perfectly as expected. However, we need to work on the slight design to make it more user-friendly. I have noticed the modal content crosses the borders at around `868px` and `590px` screen sizes. Kindly consider changing the height to ensure that all the contents remain with in the card at all screen sizes ☺️ 👍.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/226073745-58c31219-8f48-4680-bcb9-faa2405c9c8f.png)

![image](https://user-images.githubusercontent.com/99394374/226073755-7b7ed612-26d8-4c4f-acc9-01db711a65e9.png)